src/components/Text/src/Text.vue

```diff
@@ -118,7 +118,7 @@ export default {
 }
 
 .Paragraph {
-	color: var(--color-800);
+	color: var(--color-text);
```

Comment:
Same here with text color. Though I do see the utility of a `--color-text` variable that can be used on the base theme wrapper for global background and color.

Reply:
That's primarily why I set that up. I also just made some changes here to reflect other ways this could be used between both the heading and text property: https://github.com/square/maker/pull/177/commits/0f525951a7f95edf28318eae8b973d3fe0669de8

My suggestion is that all text against the background should use the same text color by default, so that would update things like radio / checkboxes:
![CleanShot 2021-11-12 at 16 13 35](https://user-images.githubusercontent.com/129122/141597649-9d2b6e98-cbfd-4e7e-ae43-e71225019d0e.png)

With something like the calendar which has both a heading (Date Year) and the cells it seems more appropriate as well to leverage the heading and text color defined in the profile vs a neutral set.
![CleanShot 2021-11-12 at 16 13 41](https://user-images.githubusercontent.com/129122/141597700-e25c5563-ff64-458e-a597-d576c38f20f6.png)